packages/client-core/src/common/services/ThemeService.tsx

```diff
@@ -56,9 +63,14 @@ const lightTheme: CSSClasses = {
   'bg-primary': '#F5F5F5',
   'bg-secondary': '#FFFFFF',
   'bg-highlight': '#D9D9D9',
+  'bg-accent': '#375DAF',
```

Comment:
There are so many duplicates here. Do we really need so many with the same colour?

Reply:
good catch that i didn't realized, fixed removed the duplicates